README.md

```diff
@@ -77,6 +77,12 @@ To turn off `animation`:
 $ adb shell am broadcast -a io.appium.settings.animation --es setstatus disable
 ```
 
+Set particular locale:
```

Comment:
I'd also mention the link to the list of all acceptable locale and language names.